build/android/ndk.version

```diff
@@ -0,0 +1 @@
+22.1.7171670
```

Comment:
Thanks I've been meaning to do something like this for a while. The version number was in too many places.